src/main/java/de/retest/recheck/ui/descriptors/Attribute.java

```diff
@@ -10,7 +10,7 @@
 
 	private static final long serialVersionUID = 1L;
 
-	public static final double IGNORE_WEIGHT = 0.0d;
+	public static final double IGNORE_WEIGHT = Float.MIN_NORMAL;
```

Comment:
Could you explain this change, commit message isn't very helpful? And isn't there also a `Double` variant?